kitsune/notifications/tests/test_api.py

```diff
@@ -38,3 +47,68 @@ def test_cant_register_for_other_users(self):
         eq_(serializer.errors, {
             'creator': ["Can't register push notifications for another user."],
         })
+
+
+class TestNotificationSerializer(TestCase):
+    def test_correct_fields(self):
+        follower = profile()
+        followed = profile()
+        q = question(creator=followed.user, save=True)
+        follow(follower.user, followed.user)
```

Comment:
Because of that giant signal handler that should be a celery task, creating the action below will create a Notification, because `follower` is following `followed`. The idea is that an `Action` object represents something a user did. But in order to track read/unread and who should be notified, `Notification` objects are created, one for every following user.

This test probably relies on too many things, and I should make the Notification object directly instead of trying to get one from the Action system.